src/templates/governance-page/DaoGovernanceDetails/DaoVotes.tsx

```diff
@@ -0,0 +1,93 @@
+import { FC, useEffect, useState } from "react"
+import { gql, request } from "graphql-request"
+import BN from "bignumber.js"
+import dayjs from "dayjs"
+import { PageSection, ResponsiveStack } from "../../../components/PageSection"
+import { H4 } from "../../../components"
+import VoteResultsCard from "./VoteResultsCard"
+import { Proposal, SNAPSHOT_URL, SnapshotProposal } from "./types"
+
+const DaoVotes: FC = () => {
+  const [proposals, setProposals] = useState<Proposal[]>([])
+
+  const getProposals = async () => {
+    const query = gql`
+      {
+        proposals(
+          first: 2
+          skip: 0
+          where: { space_in: ["threshold.eth"] }
+          orderBy: "created"
+          orderDirection: desc
+        ) {
+          id
+          author
+          title
+          link
+          start
+          end
+          state
+          choices
+          scores
+        }
+      }
+    `
+    const result: { proposals: SnapshotProposal[] } = await request(
+      SNAPSHOT_URL,
+      query
+    )
+
+    const scrubbedProposals = result.proposals.map((proposal) => {
+      const dateRange = `${dayjs(proposal.start * 1000).format(
+        "MMM D YYYY"
+      )} - ${dayjs(proposal.end * 1000).format("MMM D YYYY")}`
```

Comment:
Also maybe the `VoteResultCard` component should handle formatting and we should just return `start` and `end` date in a proposal object. I see a case where we would like to calculate how many days are left until the end of voting, and we will not be able to do this because we have a formatted string (Actually, we would be able to do this, but it would be tricky because of converting the `dateRange` to 2 dates).